api/src/main/java/org/openmrs/api/impl/ConditionServiceImpl.java

```diff
@@ -114,6 +114,20 @@ public Condition getCondition(Integer conditionId) {
 		return conditionDAO.getActiveConditions(patient);
 	}
 
+	/**
+	 * Gets all conditions of patient
+	 *
+	 * @param patient - the patient to retrieve conditions for
+	 *                   
+	 * @return a list of the patient's conditions
```

Comment:
Instead of having a duplicate of the javadoc here, you would use the @see annotation to point to the one at the service interface.

Reply:
@dkayiwa  I updated the PR